package stringtie 1.2.3 fails to install on Main

Any help appreciated.

```
In file included from bgzf.c:36:
bgzf.h:29:18: error: zlib.h: No such file or directory
bgzf.c: In function ‘open_write’:
bgzf.c:186: error: ‘Z_DEFAULT_COMPRESSION’ undeclared (first use in this function)
bgzf.c:186: error: (Each undeclared identifier is reported only once
bgzf.c:186: error: for each function it appears in.)
bgzf.c: In function ‘deflate_block’:
bgzf.c:297: error: ‘z_stream’ undeclared (first use in this function)
bgzf.c:297: error: expected ‘;’ before ‘zs’
bgzf.c:298: error: ‘zs’ undeclared (first use in this function)
bgzf.c:305: warning: implicit declaration of function ‘deflateInit2’
bgzf.c:305: error: ‘Z_DEFLATED’ undeclared (first use in this function)
bgzf.c:306: error: ‘Z_DEFAULT_STRATEGY’ undeclared (first use in this function)
bgzf.c:307: error: ‘Z_OK’ undeclared (first use in this function)
bgzf.c:311: warning: implicit declaration of function ‘deflate’
bgzf.c:311: error: ‘Z_FINISH’ undeclared (first use in this function)
bgzf.c:312: error: ‘Z_STREAM_END’ undeclared (first use in this function)
bgzf.c:313: warning: implicit declaration of function ‘deflateEnd’
bgzf.c:345: warning: implicit declaration of function ‘crc32’
bgzf.c: In function ‘inflate_block’:
bgzf.c:371: error: ‘z_stream’ undeclared (first use in this function)
bgzf.c:371: error: expected ‘;’ before ‘zs’
bgzf.c:373: error: ‘zs’ undeclared (first use in this function)
bgzf.c:380: warning: implicit declaration of function ‘inflateInit2’
bgzf.c:381: error: ‘Z_OK’ undeclared (first use in this function)
bgzf.c:385: warning: implicit declaration of function ‘inflate’
bgzf.c:385: error: ‘Z_FINISH’ undeclared (first use in this function)
bgzf.c:386: error: ‘Z_STREAM_END’ undeclared (first use in this function)
bgzf.c:387: warning: implicit declaration of function ‘inflateEnd’
bgzf.c: In function ‘check_header’:
bgzf.c:405: error: ‘Z_DEFLATED’ undeclared (first use in this function)
make[1]: *** [bgzf.o] Error 1
make: *** [samtools-0.1.18/libbam.a] Error 2
```

--- a/packages/package_stringtie_1_2_3/tool_dependencies.xml
+++ b/packages/package_stringtie_1_2_3/tool_dependencies.xml
@@ -1,9 +1,17 @@
 <?xml version="1.0"?>
 <tool_dependency>
+    <package name="zlib" version="1.2.8">
+        <repository name="package_zlib_1_2_8" owner="iuc" prior_installation_required="True" />
+    </package>
     <package name="stringtie" version="1.2.3">
-      <install version="1.0">
+        <install version="1.0">
             <actions>
                 <action type="download_by_url" sha256sum="1a323fd17c24889748c645b3dfe5a06305236d9899e87a27939a68404d2d9d09">https://ccb.jhu.edu/software/stringtie/dl/stringtie-1.2.3.tar.gz</action>
+                <action type="set_environment_for_install">
+                    <repository name="package_zlib_1_2_8" owner="iuc">
+                        <package name="zlib" version="1.2.8" />
+                    </repository>
+                </action>
                 <action type="shell_command">make release</action>
                 <action type="move_file">
                     <source>stringtie</source>

